### DISCUSSION TITLE
How to remove a (sub-)dataset from source and target while keeping replication happy

### DISCUSSION THREAD

**Discussion by luqasn** [psy0rz/zfs_autobackup#272] [Q&A]
Hi all!

What a wonderful tool this is, I use it to make an offsite backup of my server and it works well, even through forced disconnects of the ISP and such.
But how do I properly remove a (sub-)dataset from the replication without getting errors on the next sync?
So I have the replication tag on `/tank/` and the children inherit it. So `/tank/somejunk/random` gets replicated as well. Now I want to remove that folder altogether (on source and backup target).
What I did was just zfs destroy it, but then all following runs of zfs_autobackup complained about differences in the snapshots of source and target.
What's the proper way to do this?

Thanks a lot!

**Comment by psy0rz** [comment]
What kind of errors? If you destroy them on both sides it should be fine.

Zfs-autobackup does keep some hidden state or db somewhere. Only the autobackup: property is important.
